static/app/views/alerts/rules/issue/sentryAppRuleModal.spec.tsx

```diff
@@ -233,10 +233,74 @@ describe('SentryAppRuleModal', function () {
       const complexityInput = screen.getByLabelText('What is the estimated complexity?', {
         selector: 'input#complexity',
       });
+
+      expect(screen.queryByText('Low')).not.toBeInTheDocument();
       await userEvent.click(complexityInput);
       expect(screen.getByText('Low')).toBeInTheDocument();
       expect(screen.getByText('Medium')).toBeInTheDocument();
       expect(screen.getByText('High')).toBeInTheDocument();
     });
+
+    it('should populate skip_load_on fields with the default value', async function () {
```

Comment:
Is there a test that shows this doesn't affect other fields?

Reply:
Q: err can you elaborate a bit more on the case? I'm not really following

Reply:
It's not very elaborate which is why I approved and left it to you to decide. When I write tests I always write one for the happy path which I have in mind and one to say my code doesn't affect other things that it should not affect. Not sure if it's applicable here.

For example, should there be a test for a field without skip_on_load if the behavior there is different?

Reply:
Ohh I see, yeah like the original tests didn't test a non skip_on_load select so I could make one for that.